prism/src/main/java/org/apache/falcon/resource/proxy/ExtensionManagerProxy.java

```diff
@@ -94,14 +94,16 @@
 
     //SUSPEND CHECKSTYLE CHECK ParameterNumberCheck
     @GET
-    @Path("list/{extension-name}")
+    @Path("list")
     @Produces({MediaType.TEXT_XML, MediaType.APPLICATION_JSON})
     public ExtensionJobList getExtensionJobs(
-            @PathParam("extension-name") String extensionName,
+            @QueryParam("extension-name") String extensionName,
```

Comment:
Why query param? Path param is REST compliant and is in-line with other APIs too.